src/main/java/com/cinema/dao/impl/MovieDaoImpl.java

```diff
@@ -21,13 +22,12 @@ public Movie add(Movie movie) {
             transaction = session.beginTransaction();
             Long movieId = (Long) session.save(movie);
```

Comment:
we don't use this variable anymore - let's save without creating it

Reply:
That makes perfect sense, of course)) Silly me)